ROADMAP.md

```diff
@@ -0,0 +1,51 @@
+# Roadmap
+
+We track upcoming work on Azure Service Operator (ASO) using [GitHub Issues](https://github.com/Azure/azure-service-operator/issues?q=is%3Aissue+is%3Aopen).
+
+## Release Cadence
+
+We target a new release of ASO approximately every two months, though this may vary depending on the amount of work we have planned as well as external factors.
+
+Our current release plan:
+
+| Version | Estimated Release |
+| ------- | ----------------- |
+| [2.6.0](https://github.com/Azure/azure-service-operator/issues?q=is%3Aopen+is%3Aissue+milestone%3Av2.6.0) | Mid February 2024 |
+| [2.7.0](https://github.com/Azure/azure-service-operator/issues?q=is%3Aopen+is%3Aissue+milestone%3Av2.7.0) | Mid April 2024 |
+| 2.8.0 | Mid June 2024 |
+| 2.9.0 | Mid August 2024 |
+
+Where linked, versions go to the list of open issues for that release.
+
+Any items note completed in time for one release will be carried over to the next, and may result in us pushing other items to a later release.
+
+Partway through a release cycle, we'll review the list of issues assigned to the upcoming release and redistribute issues as needed.
+
+If you're waiting on a particular resource or feature to be released, please comment on the relevant issue (or create a new issue) to let us know. We'll do our best to keep you updated on progress.
+
+## Experimental Releases
+
+We publish an [experimental release](https://github.com/Azure/azure-service-operator/releases/tag/experimental) on a regular basis, suitable for testing and other non-production use cases. Such releases may not be stable and should not be used in production.
+
+## Prior Releases
```

Comment:
This is pretty far down, can we move it before the  "planning" section?

Reply:
This document isn't the canonical list of prior releases, it's a forward looking roadmap. 

I very nearly didn't include this section at all because it's backward looking, but figured it would be useful as a reference. I don't think it belongs earlier in the doc.